src/Controller/HealthController.php

```diff
@@ -40,8 +41,11 @@ public function __construct(
     #[Route('/internal/health', name: 'monitor.internal_health', methods: ['GET'])]
     public function __invoke(): JsonResponse
     {
-        $statusResponse = $this->healthChecker->check();
-        
+        try {
+            $statusResponse = $this->healthChecker->check();
+        } catch (\Exception $exception) {
```

Comment:
1. Suggestion for improvement: catch the more generic `Throwable`?
2. Nitpick. Import the Exception or Throwable. We do not want the `/FQN` notation. In this case it is very brief as it does not reside in a length namespace..